feature-libs/product-configurator/rulebased/occ/variant/variant-configurator-occ.models.ts

```diff
@@ -19,12 +19,25 @@ export namespace OccConfigurator {
   }
 
   export interface Prices {
-    configId?: string;
+    configId: string;
+    attributes?: Supplements[];
     pricingError?: boolean;
     showDeltaPrices?: boolean;
     priceSummary?: PriceSummary;
   }
 
+  export interface Supplements {
+    csticUiKey: string;
+    selectedValues: string[];
+    priceSupplements: ValueSupplements[];
+  }
+
+  export interface ValueSupplements {
```

Comment:
ValueSupplements -> ValueSupplement
Is it correct that we use plural here?
The same is valid for Supplements above.